### ISSUE TITLE
 "have" as an alternate opener?

### ISSUE THREAD

**Issue by Vermoot** [aerickt/aerick-phrasing#7]
I think it would be useful to have "have" as an additional alternate starter, for phrases such as "have you remembered the".

I'm not sure what the unique starter would be for that, the obvious `SR` would certainly conflict with lots of words, with it also being a starter sound.

Thoughts?

**Comment by aerickt** [comment]
Yeah it can definitely be added. Would just need to look for a starter that doesn't conflict.

From https://discord.com/channels/136953735426473984/136953809409802240/928063506216009779

> SKPWH (1), SKPWHR (1), STPWR (1), TKWH (1), SKWH (2), TWHR (2), STKPWH (3), STKPHR (3), STKPR (3), KPWH (4), STPW (4), TKWHR (4), STKW (5), SWHR (5), KWH (6), SKWHR (6), STKHR (6), STKWR (7), STPWH (7), STW (7), SKPHR (8), SKPR (8), STKPWR (9), STKPWHR (10), SPWH (10), TWR (10), SWR (11), SKH (12), STPHR (12), TPWH (12), STKPH (14), TWH (15), STHR (16), KPWR (17), KPWHR (17), SKPW (19), SKPWR (20), SWH (20), KWHR (21), SPWHR (22), SKPH (23), TPWHR (24), STPR (29), STKP (29), TKPWH (31), TPWR (35), TKH (37), WHR (38)...
here's a list of starters and their number of occurences (starters with 0 occurences not included) 

Starters without any uses at all:
['STKH', 'STWH', 'STKWH', 'STWHR', 'STKWHR', 'STPWHR']

I don't use the alternate starters myself, actually, so I'm not sure what would be a good chord. What do you think about `KWH`? It's relatively easy to stroke and is only used in misstrokes as far as I can tell:
```
cat main.json | grep "\"KWH[AOEU]"
"KWHA": "what",
"KWHAFR": "whatever",
"KWHATS": "what's",
"KWHO": "who",
"KWHOEL": "whole",
"KWHEPB": "when",
```

**Comment by Vermoot** [comment]
Yeah, `KWH` seems good!

I think the list you've been looking at might not include `A` and `O` as part of the starters though, so there might be more available for alternate starters. Maybe even a number key as well, since it's used only in middles, which are unavailable with alternate starters?

Which is a good thing, since I'm sure we'll find more worthy phrases to add... right now I'm thinking of "didn't", for example. Or more pronouns to add with alternate starters ("she" being excluded feels especially weird)

**Comment by aerickt** [comment]
Yeah so the thing with the alternate starters is that they were just a way to squeeze more phrases into the system when Jade first made the system. That's why a lot of pronouns have been left out.

But just yesterday I realised that with the extra `-F` medial there's enough room to include the rest of the pronouns:
```
            "WHAE":"what he",
            "WHAU":"what you",
            "WHAEU":"what I",
            "WHAEF":"what she",
            "WHAUF":"what they",
            "WHAEUF":"what we",
            "WHAF": "what it",

            "SKPWE":"doesn't he",
            "SKPWU":"don't you",
            "SKPWEU":"don't I",
            "SKPWEF":"doesn't she",
            "SKPWUF":"don't they",
            "SKPWEUF":"don't we",
            "SKP-F":"doesn't it",

            "STKOE":"does he",
            "STKOU":"do you",
            "STKOEU":"do I",
            "STKOEF":"does she",
            "STKOUF":"do they",
            "STKOEUF":"do we",
            "STKOF":"does it",

            "STKPWOE":"did he",
            "STKPWOU":"did you",
            "STKPWOEU":"did I",
            "STKPWOEF":"did she",
            "STKPWOUf":"did they",
            "STKPWOEUF":"did we",
            "STKPWOF":"did it",

            "STHAE":"that he",
            "STHAU":"that you",
            "STHAEU":"that I",
            "STHAEF":"that she",
            "STHAUF":"that they",
            "STHAEUF":"that we",
            "STHAF":"that it",

            "STPAE":"if he",
            "STPAU":"if you",
            "STPAEU":"if I",
            "STPAEF":"if she",
            "STPAUF":"if they",
            "STPAEUF":"if we",
            "STPAF":"if it",

            "SWHE":"when he",
            "SWHU":"when you",
            "SWHEU":"when I",
            "SWHEF":"when she",
            "SWHUF":"when they",
            "SWHEUF":"when we",
            "SWHF":"when it",

            "KWHE":"has he",
            "KWHU":"have you",
            "KWHEU":"have I",
            "KWHEF":"has she",
            "KWHUF":"have they",
            "KWHEUF":"have we",
            "KWH-F":"has it"
```

I think `A` and `O` can be used as medials like for 'really' or 'not' if the starters don't use them.

**Comment by Vermoot** [comment]
That seems great!

> I think A and O can be used as medials like for 'really' or 'not' if the starters don't use them.

I was just thinking for "what" and "when", `A` and `O` could be used for "did" and "do".

`WH  -> what`  
`WHA -> what did`  
`WHO -> what do`  

`SWH  -> when`  
`SWHA -> when did`  
`SWHO -> when do`

**Comment by Vermoot** [comment]
I'll try them out and report back, but if your new dictionaries are slated to replace this one I guess it'll be more relevant to apply those changes there. I'll be happy to help in any way I can on that!

**Comment by YannCebron** [comment]
@aerickt could you please give an update regarding your plans with replacing this system? Thank you!

**Comment by aerickt** [comment]
Apologies for the late reply, but I've given up on trying to improve this dictionary as I was unhappy with the code and I also didn't feel any additions would result in any increased speed or comfort. Since then, Tharelia#2062 on Discord has come up with their own phrasing system which I've actually switched to:
https://github.com/jthlim/jeff-phrasing

It lacks some medials such as "really", but I think its potential is far greater than my phrasing system. I haven't had the time to fully explore it, but it's a great option to try out.
